CONTRIBUTING.md

```diff
@@ -139,9 +111,8 @@ If you're writing a longer block of text to a terminal, wrap it at 80 characters
    Endpoint: /v1/status
 ```
 
+### License
 
-**License**
-
-If you make a new source file, you must copy the license preamble from an existing file to the top of it. We can't merge a PR with unlicensed source files. We also can't merge PRs unless all authors have signed the [Contributor License Agreement](https://cla-assistant.io/loadimpact/k6).
+If you make a new source file, you must copy the license preamble from an existing file to the top of it. We can't merge a PR with unlicensed source files. We also can't merge PRs unless all authors have signed the [Contributor License Agreement](https://cla-assistant.io/grafana/k6).
```

Comment:
This is no longer needed so I think we can drop the whole section

Reply:
:facepalm: true!